app/models/link_check_report.rb

```diff
@@ -0,0 +1,18 @@
+class LinkCheckReport
+  include Mongoid::Document
+  include Mongoid::Timestamps
+
+  embeds_many :links
+
+  accepts_nested_attributes_for :links
+
+  field :batch_id, type: Integer
+  field :status, type: String
+  field :link_reportable_type, type: String
+  field :link_reportable_id, type: Integer
```

Comment:
Here https://github.com/alphagov/manuals-publisher/pull/1225/files#diff-4c62e07f8da55a84ff899aa3c1293d9eR26 we see to be referring to some a `manuals_id` and `section_id` field, do we need to change that here too?

Reply:
As mentioned on that PR, if we're going to only have `manual_id` and `section_id`, we could define a method:

```
def is_for_section?
  section_id.present?
end
```